src/main/java/org/terasology/gookeeper/system/BreedingSystem.java

```diff
@@ -60,6 +60,10 @@
 @Share(value = BreedingSystem.class)
 public class BreedingSystem extends BaseComponentSystem {
 
+    // mapping of parent color portion combination to HTML color names used in gooey assets
+    private static final ImmutableMap<String, String> COLORS;
+    private static final Logger logger = LoggerFactory.getLogger(BreedingSystem.class);
```

Comment:
I would recommend to put the logger at the top-most position wherever possible as it doesn't really have to do anything with the actual logic of the class.

```suggestion
    private static final Logger logger = LoggerFactory.getLogger(BreedingSystem.class);
    private static final ImmutableMap<String, String> COLORS;
```